recipes/pantax/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+if [ "$(uname)" == "Darwin" ]; then
+    cp -rf $SRC_DIR $PREFIX
+else
+    cd ${SRC_DIR}/scripts
+    chmod +x pantax
+    chmod +x data_preprocessing
+    cp ${SRC_DIR}/scripts/pantax ${SRC_DIR}/scripts/data_preprocessing ${PREFIX}/bin
+    cp ${SRC_DIR}/scripts/*py ${PREFIX}/bin
+
+    mkdir -p ${PREFIX}/bin/tools
+    cd ${SRC_DIR}/tools/fastix
+    cargo install fastix --root ./
```

Comment:
Usually this command looks like `RUST_BACKTRACE=1 cargo install --verbose --locked --no-track --root $PREFIX --path .` This way it will install the binary to `$PREFIX/bin`

Reply:
Sorry, I am not very familiar with the Rust. I have already made modifications.